pkgs/by-name/av/avisynthplus/package.nix

```
@@ -0,0 +1,43 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  gitUpdater,
+  fetchpatch,
+}:
+
+stdenv.mkDerivation (
+  finalAttrs: {
```

Was this because of formatter?
```suggestion
stdenv.mkDerivation (finalAttrs: {
```

---

yeah the rfc formatter did that.